project/Dependencies.scala

```diff
@@ -81,7 +81,7 @@ object Dependencies {
     // jaxb-api needed by WorkspaceApiServiceSpec.bagitService() method
     "javax.xml.bind"                 % "jaxb-api"            % "2.3.1"   % "test",
     // provides testing mocks
-    "com.google.cloud"               % "google-cloud-nio"    % "0.123.7" % "test",
+    "com.google.cloud"               % "google-cloud-nio"    % "0.123.16" % "test",
```

Comment:
this provides the `LocalStorageHelper` class used in tests, related to the google2 GCS libraries; it felt ok to upgrade it in sync with the google2 upgrade.